### DISCUSSION TITLE
Catalyst DA2 receiver & QFIELD

### DISCUSSION THREAD

**Discussion by Madmot454** [opengisch/QField#4219] [Q&A]
Hello,

I'm new with GNSS receiver (Catalyst DA2). I have a 1cm precision subscription. I've been testing my receiver two nights in a row with a qfield project base on EPSG:2950 (NAD83)(SCRS) MTM zone commonly use here in Quebec (montreal, Quebec, Canada) (I think every road work uses this system)

When activated, my Trimble Mobile Manager is outputing NAD83(SCRS) (epoch1997) coordinated on basis (auto and same as the source). Everything looks good on the app when testing everything on geodesic marker around town. The coordinate given by the natural ressources ministry are EPSG:22248 NAD83(CSRS)v2 / SCoPQ zone 8 ,
Everything is fine until I comme back at the office : I noticed small errors in X and Y. My Z is perfect, right on the CGVD2013 standard as setup in Qfield. The real problem is : I usually have a roughly 0.1 error in X and a 1m error in Y. 

What is my problem? Do I make an error when creating my qfield project? Wrong coordinate system? Tonight, I'll try a project in ESPG:22248, the same has given by the governement. Thats to coordinate I'm trying to be as close as possible. Oh! they also gives UTM coordinates also in the NAD83 (SCRS) V2 standard.

I will also try to look in the state mode of my Trimble manager to see if the coordinates given by GNSS output is right and in order with the natural ressources minitry. 

And for you guys, what are your setting in your GField projects for a survey? Do you guys stay in the coordinate system mostly use locally or in the region or you try to be as close as the output given by your GNSS receivers - the mock location service given by the phone when connected to a GNSS antenna or receiver?

Thanks and excuse my french!

Math  

### Desktop (please complete the following information)
 - OS: [e.g. Windows 10]
 - QGIS Version [e.g. 3.28.0]

### Mobile (please complete the following information)
 - Device: [e.g. Samsung S22]
 - OS: [e.g. Android :13]
 - QField version: [2.7.5]

**Comment by nirvn** [comment]
@Madmot454 , I've converted this into a discussion for now, feels more appropriate. 

Have you made any more progress on this? We have a couple of QField users active on github that are from Quebec, let's hope they see this call for help and come through :)

I assume your Trimble receiver feeds in WGS84 lat, lon, so the problem is likely having to do the transformation between WGS84 coordinates into your project CRS. Have you checked the _Transformations_ tab within the project properties dialog (in QGIS) to see if it uses the most accurate option?

**Comment by Madmot454** [comment]
Hello, 
 Thanks for taking time to answer my question. 
Yes, I fact, I made some progress. My Qgis/Qfield Project was in the wrong CRS. I did a test with a new project in ESPG: 22248, witch is the NAD83 (SCRS) V2 exactly as the one the Ministry of natural ressources uses. (or maybe I'm wrong again 'cause I found this info given by the government (https://mrnf.gouv.qc.ca/wp-content/uploads/CO_codes_epsg_quebec.pdf). I just found that here around Montreal (zone 8), maybe I should have use 32188? 
Anyways, with my Catalyst DA2 receiver, Qfield was giving me exactly the position I wanted in 22248. That QFIELD software is a beast! So I realise my problem was that I was in the wrong CRS the whole time. I was in ESPG: 2259 witch is close but not on it. It's strange cause most of our project are in that format : NAD83 (SCRS) MTM zone 8.... 
Anyways, now, I'm able to localise myself precisely in accordance with the grid of the Quebec governement and I can create my own geodesic stations everywhere around town, in the fields and in the woods.  

The DA2 receiver can feed a postion in a variety of CRS. You can make a quick selection between 1-same as the source (no transformation), 2-Auto, 3-Select from a list. Both 1 and 2 will feed NAD83 (SCRS) v7 (Epoch:1997). 3- gives you a list of close to 30 system including many ETRSs, EUREFs, ITRFs, JDGs, MAGNAs, NAD83s, WGS84 (G1762) and many more. 

Thanks again!
